lib/generators/active_record/session_migration_generator.rb

```diff
@@ -19,6 +19,10 @@ def session_table_name
           end
           current_table_name
         end
+
+        def migration_version
+          "[#{ActiveRecord::VERSION::MAJOR}.#{ActiveRecord::VERSION::MINOR}]" if ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
There's a short-hand for this:

~~~ruby
def migration_version
  "[#{ActiveRecord::Migration.current_version}]" if ActiveRecord::Migration.respond_to?(:current_version)
end
~~~